packages/crud/test/dom/__snapshots__/crud.test.snap.js

```diff
@@ -6,9 +6,6 @@ snapshots["vaadin-crud host default"] =
   editor-position=""
   fullscreen=""
 >
-  <h3 slot="header">
-    Edit item
-  </h3>
```

Comment:
This node is teleported, alongside with `vaadin-crud-form` element and buttons below. The reason is that now we set `_form` earlier and `_fullscreen`  breakpoint forces the teleporting to happen.